src/mongo_logger.ts

```diff
@@ -430,10 +430,30 @@ export class MongoLogger {
   maxDocumentLength: number;
   logDestination: MongoDBLogWritable | Writable;
 
+  /**
+   * This method should be used when logging errors that do not have a public driver API for
+   * reporting errors.
+   **/
   error = this.log.bind(this, 'error');
+  /**
+   * This method should be used to log situations where undesirable application behaviour might
+   * occur. E.g.: encountering an unrecognized option in a connection string
```

Comment:
```suggestion
   * occur. For example, encountering an unrecognized option in a connection string.
```

We can just complete the sentence. Also this example doesn't apply to our driver since we error on unrecognized options